rules/nat_test.go

```diff
@@ -54,6 +54,26 @@ var _ = Describe("NAT", func() {
 			},
 		}))
 	})
+	It("should render rules when active with explicit port range", func() {
```

Comment:
Name of test case is incorrect after backport (we don't have port ranges in v3.0)